shell/common/shell_unittests.cc

```diff
@@ -1014,5 +1015,114 @@ TEST_F(ShellTest, Screenshot) {
   DestroyShell(std::move(shell));
 }
 
+static bool MemsetPatternSetOrCheck(uint8_t* buffer,
```

Comment:
Add a short doc comment, since it's a utility. Particularly, document that if `set_or_check`is true, you set, otherwise you check.

Reply:
Done. Made the bool an enum.